sqlmesh/core/dialect.py

```diff
@@ -1214,6 +1214,9 @@ def extract_audit(v: exp.Expression) -> t.Tuple[str, t.Dict[str, exp.Expression]
     elif isinstance(v, exp.Func):
         func = v.sql_name()
         args = list(v.args.values())
+    elif isinstance(v, exp.Tuple):  # airflow only
```

Comment:
Why is this in the `extract_audit` function?

Reply:
I see that audits are now handled the same way as signals.
1. The function name is obsolete
2. Can we apply this logic to signals only? Otherwise, we now allow unexpected syntax for audits